src/screens/InputUsername/index.js

```diff
@@ -312,7 +312,7 @@ const ChooseUsername = () => {
             </View>
             <View style={styles.containerInfo}>
               <View style={styles.parentIcon}>
-                <IconWarningCircleBlue fill={COLORS.white2} fillIcon={COLORS.signed_primary} />
+                <IconWarningCircleBlue fill={COLORS.white} fillIcon={COLORS.signed_primary} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the `IconWarningCircleBlue` component has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change is intentional and does not affect the visibility or aesthetics of the icon in different themes or backgrounds.
```diff
- <IconWarningCircleBlue fill={COLORS.white2} fillIcon={COLORS.signed_primary} />
+ <IconWarningCircleBlue fill={COLORS.white} fillIcon={COLORS.signed_primary} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->